### ISSUE TITLE
[DISCUSSION] Future for this project.

### ISSUE THREAD

**Issue by divy-work** [dan-divy/spruce#61]
So its been a few months since we've commited something to this projects and the lack of time and dedication required is clearly an indication that the project will not be developed actively anymore. So I am planning to either archive spruce or look out for maintainers.

A few points on why archiving is better:
* We all can agree on the fact that Spruce is not efficient and lacks code quality.
* We are not compliant to the new tech standards for example: using EJS, callbacks (instead of promises)
* Filled with bugs 

Would like to know your thoughts or if anyone of you are willing to maintain this project.

(cc @dan-divy/creators)

**Comment by dan-online** [comment]
@divy-work I completely agree, we should probably consider nuxt or something similar as I think we can agree EJS is not the best. A v2 branch would be good and we can start stripping anything excessive.

**Comment by GrosSacASac** [comment]
What are the disadvantages of  EJS if I may ask

**Comment by littledivy** [comment]
@GrosSacASac It's not a disadvantage, but it's an overhead rendering all of the page at once. 
For example, we're currently rendering all the posts on the index page at once which increases server load. This can be avoided by lazy loading components.

**Comment by anosh-anosh** [comment]
I agree, also it is hosted on glitch. I prefer heroku hosting as it is faster. I posted and commented twice and it went down. definetley in need of some code optimisation and maybe a new library and a v2.
